src/main/resources/anniv5Resources/localization/eng/dimensiongatepack/Keywordstrings.json

```diff
@@ -4,6 +4,6 @@
     "NAMES": [
       "hoard"
     ],
-    "DESCRIPTION": "When played, reduce Hoard by #bX (the card's cost). When Hoard reaches #b0, its effects trigger."
+    "DESCRIPTION": "When played, if this card's cost is greater than 0 and lower than it's Hoard: NL Reduce Hoard by the card's cost and ignore all other card effects. NL Return the card to your hand. NL The card #yRetains this turn."
```

Comment:
I do not like this iteration of the keyword at all. It's a mechanics overload that tries to cover all angles but makes the card less parseable as a result.

I definitely understand why these changes were made after you watched Frost being confused but I think you have to keep in mind that he's dyslexic and a streamer (baiting donations, looking at chat instead of taking a second, etc)

Retain and Persist are disjointed here and should not be part of this already massive keyword. If you can't stomach putting those keyword on the card due to complexity, then maybe the effect need a bit more of an overhaul.
Also, the new wording o the base effect is even more confusing than it's first iteration `if this card's cost is greater than 0 and lower than it's Hoard` I definitely needed a few rereads to get this because by covering all angles you made the sentence hard to parse.
Simplicity is important and this really misses the mark.